docker/worker/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM opensuse:13.2
 MAINTAINER Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>
 
-RUN useradd -m geekotest && \
-    zypper --non-interactive --gpg-auto-import-keys in ca-certificates-mozilla curl && \
-    zypper ar -f http://download.opensuse.org/repositories/devel:openQA:stable/openSUSE_13.2/devel:openQA:stable.repo && \
```

Comment:
few moments ago we released stable 4.2. It matches today's master. So you can keep using stable to avoid random devel breakage.